types/remarkable/package.json

```diff
@@ -5,13 +5,12 @@
     "projects": [
         "https://github.com/jonschlinkert/remarkable"
     ],
-    "dependencies": {
+    "dependencies": {},
+    "devDependencies": {
         "@types/highlight.js": "^9.7.0",
+        "@types/remarkable": "workspace:.",
         "highlight.js": "^9.7.0"
     },
-    "devDependencies": {
-        "@types/remarkable": "workspace:."
-    },
```

Comment:
This is definitely not correct. This is a dev dep on itself and is required. These deps are not published.

Reply:
Oh, my mistake. This diff is just really silly looking, sorry!!